src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -369,15 +369,15 @@ class TDatabaseFeedManager final : public Observer<nlohmann::json&>
     }
 
 private:
-    std::unique_ptr<TRouterSubscriber> m_contentUpdateSubscription;
-    std::unique_ptr<TContentRegister> m_contentRegistration;
     std::shared_ptr<TIndexerConnector> m_indexerConnector;
+    std::unique_ptr<TContentRegister> m_contentRegistration;
```

Comment:
If the order of the variables is important for the correct behavior of the class, I'd add a comment to prevent this problem from happening again.

Reply:
fixed